### DISCUSSION TITLE
Irregular time series

### DISCUSSION THREAD

**Discussion by Yassine97HF** [alan-turing-institute/sktime#3281] [Ideas]
Hello, i want to ask the sktime community , if there are a transformers that deals with irregular time series.

Thaks!

**Comment by fkiraly** [comment]
Yes & no!

Yes, as in, the `TSInterpolator` might do the trick (linear interpolation)

No, as in, a number of other common ways to deal with irregular data are not yet implemented, and it would be great to have them!
For instance, a simple binner https://github.com/alan-turing-institute/sktime/issues/242, help would be appreciated!

@grzegorzrut has also been working on a time delta based version of the `WindowSummarizer`.

**Reply by Yassine97HF** [reply]
Thanks for the informations 🙏

**Comment by BradKML** [comment]
Sorry for this answer, but aren't there models that adapts to this situation e.g. Prophet?
